{% extends "sunddg/base.html" %}

{% load staticfiles%}

{% block main_content %}

<style>
.supindex 
{
	vertical-align:super;
	font-size:60%;
} 
.subindex 
{
	vertical-align:sub;
 	font-size:60%;
 }
p {                 
    text-align: justify;
}               
li {                
    text-align: justify;
}                   
</style>

<h3> How to use MutaBind Server? </h3>

<h4 class="subheader">Step 1 - Provide the structure of a protein-protein complex</h4>

<p> You can either input the PDB code, the structure will be retrieved from the Protein Data Bank, or upload your own structure file, which must comply with the PDB format. </p>
<p> In either case, the structure file should contain at least two distinct protein chains. Please note that:</p>
<ul style="margin-left: 3rem">
	<li>Bioassembly (biological unit) is the default structure. You can choose all biological assemblies for each protein complex and the first bioassembly (.pdb1) will be used as default. </li>
	<li>If the structure file includes multiple NMR models, only the first one will be used in calculations.</li>
	<li>If the uploaded structure file includes multiple protein complexes, only the first one will be considered.</li>
	<li>If a protein complex is classified as protease-inhibitor, a special model optimized for the protease-inhibitor complexes can be specified in the option at the bottom of the entry page. </li>
	<!-- <li>The maximum allowed number of atoms in structure is 360720, number of residues should not exceed 120240 and the maximum number of chains is 48 for a protein complex (counting only interaction partners). </li> -->
	<li>Non-protein molecules and non-standard residues are not taken into account during the calculations.</li>
	<li>MutaBind chooses the first coordinate for the atom if several different coordinates are provided in the PDB file.</li>
</ul>

If a valid PDB code is input, the title of the PDB will be shown.

<img src="{% static 'myimgs/help1.PNG' %}" style="display:block;margin:auto;width:700px;height:auto;margin-top:2rem;margin-bottom:2rem"/>

<h4 class="subheader">Step 2 - Define two partners for a protein-protein interaction</h4>

<p>A user can define interaction partners by dragging and dropping (or clicking on triangles in left or right corner) any chain thumbnail into the box of first (Partner 1) or second partners (Partner 2). </p>

<ul style="margin-left: 3rem">
	<li>MutaBind provides a 3D view of the complex colored by chains or partners. Each chain is listed with the corresponding name of the protein.</li>
	<li>One can assign one chain or multiple chains to either Partner 1 or Partner 2, but both partners should include at least one chain; only the selected chains will be taken into account during the calculation.</li>
	<li>For bioassemblies if any protein/chain in the structure were generated by applying symmetry operations, they are depicted with the labels of alphanumeric combinations (for example, ‘E_1’) indicating the source molecule/chain from which they were generated. </li>
</ul>

<img src="{% static 'myimgs/help2.PNG' %}" style="display:block;margin:auto;width:700px;height:auto;margin-top:2rem;margin-bottom:2rem"/>

<h4 class="subheader">Step 3 - Select mutations to make a mutant protein complex</h4>

<p>One can select several mutations (amino acid substitutions) on this page but only one mutation will be introduced to the complex each time (the maximum number of single mutations is 16). Please note that when more than one mutations are selected, each of them will be treated independently.</p>
<p>After a user makes the selection of a chain and a position to be mutated, they can visualize the location of the mutated (substituted) residue in the wild-type complex in the 3D viewer.</p>

<img src="{% static 'myimgs/help3.PNG' %}" style="display:block;margin:auto;width:700px;height:auto;margin-top:2rem;margin-bottom:2rem"/>

<h4 class="subheader">Results</h4>
<p> Upon the completion of the third step and submission of the job, MutaBind provides a job identifier for checking the job status. If an e-mail address is provided, the link to the results page will be sent via e-mail. The results will be available on this page as soon as the calculations are done and they will remain on the webserver for one week.
The example below shows precalculated results for <a href="{% url 'sunddg:job' %}/2016062921493313094136716">job 2016062921493313094136716</a>.
</p>

<img src="{% static 'myimgs/help4.PNG' %}" style="display:block;margin:auto;width:750px;height:auto;margin-top:2rem;margin-bottom:2rem"/>

<p>For each protein/mutation, Mutabind provides the following results:</p>
<ul style="margin-left: 3rem">
	<li> <strong><a id="DDG" style="color:black;cursor:default"> &Delta;&Delta;G</a><span class="subindex">bind</span>(kcal mol<span class="supindex">-1</span>) </strong> is the predicted change in binding affinity induced by a mutation. A positive and negative signs correspond to destabilizing and stabilizing mutations predicted to decrease and increase binding affinity correspondingly.</li>
	<li><strong><a id="interface" style="color:black;cursor:default">Interface (yes/no)</a></strong>, MutaBind defines a residue to be located on a protein-protein interface if residue solvent accessibility in the complex is lower than in the corresponding unbound partner.</li>
	<!-- <li><strong><a id="deleterious" style="color:black;cursor:default">Deleterious (yes/no)</a></strong>, MutaBind defines a mutation as deleterious if its &Delta;&Delta;G<span class="subindex">bind</span> is higher or equal to 1.57 kcal mol<span class="supindex">-1</span>. This threshold is obtained from a ROC curve plotted on the set of experimental values for more than two thousands mutations (see below). Mutations with &Delta;&Delta;G<span class="subindex">exp</span>(kcal mol<span class="supindex">-1</span>) >= 2 are defined as deleterious. The threshold corresponds to 18% FPR and 82% TPR.
	</li> -->
	<li>
		<strong><a id="deleterious" style="color:black;cursor:default">Deleterious (yes/no)</a></strong>, MutaBind server classifies a mutation as deleterious if &Delta;&Delta;G<span class="subindex">bind</span> is higher or equal to 1.57 kcal mol<span class="supindex">-1</span>. This threshold corresponds to 18% FPR and 82% TPR which minimizes the value of &radic;<span style="text-decoration:overline">(<i>1-TPR</i>)<span style="vertical-align:+30%;" class="supindex">2</span>+
																						<i>FPR</i><span class="supindex" style="vertical-align:+30%;">2</span>)&nbsp;</span> to compensate sensitivity and specificity of retrieval.
		<img src="{% static 'myimgs/ROC_server.png' %}" style="display:block;margin:auto"/>
	</li>
	<li>
		<strong><a id="confidence" style="color:black;cursor:default">Confidence (high/low)</a></strong>, for interfacial mutations, MutaBind defines a deleterious mutation with high prediction confidence if &Delta;&Delta;G<span class="subindex">bind</span> is higher or equal to 2.24 kcal mol<span class="supindex">-1</span>, and low confience if &Delta;&Delta;G<span class="subindex">bind</span> is lower than 2.24 and higher or equal to 1.57 kcal mol<span class="supindex">-1</span>. MutaBind defines a neutral mutation with high prediction confidence if &Delta;&Delta;G<span class="subindex">bind</span> is lower or equal to 0.86 kcal mol<span class="supindex">-1</span>, and low confience if &Delta;&Delta;G<span class="subindex">bind</span> is higher than 0.86 and lower than 1.57 kcal mol<span class="supindex">-1</span>. For non-interfacial mutations, predictions have significant but limited power, so the prediction confidence for all non-interfacial mutations is defined to be low.
	</li>
	
<!-- 	<li><strong><a id="confidence" style="color:black;cursor:default">Confidence (high/low)</a></strong>, MutaBind defines a deleterious mutation with high confidence if both multiple linear regression and random forest models identify it as deleterious, and low confidence if only one of two models identifies it as deleterious.</li> -->
	<li><strong><a id="structure" style="color:black;cursor:default">Mutant PDB</a></strong>, MutaBind provides the minimized mutant structure for users to download.</li>
</ul>

<p>Results can be viewed directly on the browser or downloaded as a plain text file.</p>

<a name="ibis"></a>
<h4 class="subheader">Homologous binding sites</h4>

<p>MutaBind allows exploring homologous binding sites for each protein chain and amino acid residue that was tested. This option can be accessed by clicking on &quot;Explore&quot; link in the results table. 
</p> 
<img src="{% static 'myimgs/help5.png' %}" style="display:block;margin:auto;width:700px;height:auto;margin-top:2rem;margin-bottom:2rem"/>

<p>
    The binding sites are download from Inferred Biomolecular Interactions Server at NCBI (<a href="//www.ncbi.nlm.nih.gov/Structure/ibis/ibis.cgi">IBIS, www.ncbi.nlm.nih.gov/Structure/ibis/ibis.cgi</a>). It may take several seconds before the results are retrieved.
    IBIS server identifies protein-protein complexes and binding sites homologous to the query (protein chain that was mutated) and include the mutated site.
</p>
<p>
    Note that if there is no binding site defined for the protein chain, or if the mutated residue is not part of any binding site in IBIS no results will be displayed.
    Interactions with different partners are shown separately; see IBIS for partner description. In some cases, as in the example above, IBIS does not identify any homologs, thus only the binding site for query protein is shown.
    Otherwise, see another example below (<a href="{% url 'sunddg:job' %}/2016063019523531254256688">open precalculated job</a>), the table will show alignments of binding site residues colored according to their conservation (red denoting highly conserved positions).
    Mutated site is highlighted in yellow. One can click on a residue in a homologous structure to check it in MutaBind, which will require submitting a new job for calculation.

</p>
<img src="{% static 'myimgs/help6.png' %}" style="display:block;margin:auto;width:800px;height:auto;margin-top:2rem;margin-bottom:2rem"/>

{% endblock %}
